src/main/java/models/Team.java

```diff
@@ -0,0 +1,40 @@
+package models;
+
+import java.util.Arrays;
+
+public class Team {
```

Comment:
`Game` has a `getTeamByName` function but `Team` does not have a `getPlayerByName` function. This may or may not be useful later on for displaying, but I'd add it for consistency anyway.